.ipynb_checkpoints/experiment_and_view-checkpoint.ipynb

```diff
@@ -0,0 +1,11583 @@
+{
```

Comment:
why is this file 12 000 lines long? maybe try committing the file after clearing the output of the cells

Reply:
That is the checkpoints file, I am just going to delete it.